account_invoice_discount_lines/views/account_invoice.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <record id="invoice_supplier_form" model="ir.ui.view">
+            <field name="name">Add applied discount field to invoice line in supplier invoice</field>
+            <field name="model">account.invoice</field>
+            <field name="inherit_id" ref="account.invoice_supplier_form"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='invoice_line']/tree/field[@name='discount']"
+                       position="attributes">
+                    <attribute name="string">Discount (%)</attribute>
+                    <attribute name="name">discount_display</attribute>
```

Comment:
I'm not sure but do we need here to set a low priority for that view ? This line is changing ``discount`` into ``discount_display``, so if other modules inherit from the view and base the changes on the ``discount`` field, this will break the inheritance mechanism. don't you think ?